tests/integration/seed/usersSeed.js

```diff
@@ -216,6 +216,12 @@ const userPayload = {
   sector: userSectors[0]._id,
 };
 
+const userPayloadWithoutRole = {
```

Comment:
pourquoi tu definis ce payload dnas las seed ? tu peux directement le mettre dans ton `  it('should create user even if user not connected', async () => {`
les seed sont la pour alimenter la BDD

Reply:
done